Administrator Guide/Thin-Arbiter-Volumes.md

```diff
@@ -0,0 +1,71 @@
+# Thin Arbiter volumes in gluster
+
+Thin Arbiter is a new type of quorum node where granularity of what is
+good and what is bad data is less compared to the traditional arbiter brick.
+In this type of volume, quorum is taken into account at a brick
+level rather than per file basis. If there is even one file that is marked
+bad (i.e. needs healing) on a data brick, that brick is considered bad for
+all files as a whole. So, even different file, if the write fails
+on the other data brick but succeeds on this 'bad' brick we will return
+failure for the write.
+
+
+- [Why Thin Arbiter?](#why-thin-arbiter)
+- [Setting UP Thin Arbiter Volume](#setting-up-thin-arbiter-volume)
+- [How Thin Arbiter works](#how-thin-arbiter-works)
+
+
+# Why Thin Arbiter?
+This is a solution for handling stretch cluster kind of workload,
+but it can be used for regular workloads as well in case users are
+satisfied with this kind of quorum in comparison to arbiter/3-way-replication.
+Thin arbiter node can be placed outside of trusted storage pool i.e,
+thin arbiter is the "stretched" node in the cluster. This node can be
+placed on cloud or anywhere even if that connection has high latency.
+As this node will take part only in case of failure (or a brick is down)
+and to decide the quorum, it will not impact the performance in normal cases.
+Cost to perform any file operation would be lesser than arbiter if
+everything is fine. I/O will only go to the data bricks and goes to
+thin-arbiter only in the case of first failure until heal completes.
+
+# Setting UP Thin Arbiter Volume
```

Comment:
Is this something that gluster-ansible or, gdeploy can accomplish as well? If yes, we need a placeholder for that.